src/alto-ui/Datagrid/Datagrid.js

```diff
@@ -501,51 +482,68 @@ class Datagrid extends React.PureComponent {
               )}
             </div>
           )}
-          <div
-            role="presentation"
-            ref={this.setStaticHeaderNode}
-            onScroll={this.handleScrollXStaticHeader}
-            className={bemClass('Datagrid__header-row', { static: true })}
-          >
-            {this.renderHeaderRows(staticColumnHeaders, false, frozenColumns.length)}
-            {this.renderSummaryRow(
-              staticColumns,
-              false,
-              staticColumns.length,
-              HEADER_ROW_INDEX + 1,
-              frozenColumns.length
-            )}
+          <div ref={this.setScrollHeaderNode} className="Datagrid__header-row-container">
+            <div
+              role="presentation"
+              ref={this.setStaticHeaderNode}
+              className={bemClass('Datagrid__header-row', { static: true })}
+              style={{ width: rowsWidth }}
+            >
+              {this.renderHeaderRows(staticColumnHeaders, false, frozenColumns.length)}
+              {this.renderSummaryRow(
+                staticColumns,
+                false,
+                staticColumns.length,
+                HEADER_ROW_INDEX + 1,
+                frozenColumns.length
+              )}
+            </div>
           </div>
         </div>
+        <div
+          id={id}
+          role="grid"
+          aria-rowcount={headersCount + rows.length}
+          className={bemClass(
+            'Datagrid',
+            { loaded: true, 'with-summary': !!this.props.renderSummaryCell },
```

Comment:
That one seems to not be necessary anymore

Reply:
Good point, I wondered where is this hidden. I found the opacity: 1 :)
I will change it now